content/en/load-testing/How-Tos/intelliJ-test-run.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "How to Run Java Test Cases in intelliJ"
+linkTitle: "Running Tests in IntelliJ"
+
+weight: 500
+type: docs
+
+
+description: >
+  How to import a demo test suite and execute its java test cases in IntelliJ.
+---
+
+## Execution of Java Test Cases using IntelliJ IDE
+After downloading and installing IntelliJ on a system you can use the following process to get a demo test suite up and running on your IDE. 
+
+### Obtaining the Test Suite
+Clone the demo test suite from <a href="https://github.com/Xceptance/posters-simple-loadtest-suite" target="_blank">Github</a> to a local directory, e.g. `<posters-simple-loadtest-suite>`. 
+
+### Importing the _Posters_ Test Suite into IntelliJ 
+Once IntelliJ has started, do the following:
+1. On the left pan make sure you have selected the 'Projects' tab. 
+{{< image src="how-to/ide/intelli1.jpg" max-width="75%">}}
+.
```

Comment:
Please get rid of the `.` - the content of the image shortcode will be used as a caption! (In grey below the image in the text, and also on clicking and enlarging the picture.)